### PR TITLE
Hugin stats

### PR DESCRIPTION
**Pull request by appelskrutt34**

### CODE REVIEW

**Comment by appelskrutt34** [comment]
@marcuscvj

File: monitor/src/api/hugin.js [hunk 1]
```diff
@@ -1,6 +1,87 @@
-const endpoint = "http://localhost:8080/v1/statistics";
+const endpoint = "https://stage.xkr.mjovanc.com/api/v1/statistics/";
```
Comment (mjovanc) [inline monitor/src/api/hugin.js:1]: I think we can use the `http://localhost:8080/api/v1/statistics` since it runs on the same server.

**Review by mjovanc — CHANGES_REQUESTED** [review]
Need to fix this before.

File: monitor/src/api/hugin.js (hunk 1 above)
Comment (appelskrutt34) [reply monitor/src/api/hugin.js:1]: hmm no becuase this will be called from clients aswell

**Pull request merged**